src/sparkx/Particle.py

```diff
@@ -506,7 +506,7 @@ def pdg(self):
         -------
         pdg : int
         """
-        return self.data_[9]
+        return int(self.data_[9])
```

Comment:
Is the casting to int really necessary? Aren't data types set with initialization?
If this is the case, this cast would be obsolete and I would rather not do it as it consumes computational time

Reply:
This seems to be needed. I tried without the cast and it returned a 211.0 for a pion.